images/csi-provisioner.yml

```diff
@@ -18,6 +18,6 @@ labels:
   io.k8s.display-name: External provisioner for CSI volumes
   io.openshift.tags: csi,storage,provisioner
   vendor: Red Hat
-name: openshift/ose-csi-provisioner
+name: openshift/ose-csi-external-provisioner
```

Comment:
wasn't this disabled?  or just in 4.1?

Reply:
i have no idea. the image exists and is building on 4.0.